catalog/core/catalog-core-camelcomponent/src/main/resources/OSGI-INF/blueprint/blueprint.xml

```diff
@@ -15,32 +15,34 @@
            xmlns="http://www.osgi.org/xmlns/blueprint/v1.0.0"
            xsi:schemaLocation="
   http://www.osgi.org/xmlns/blueprint/v1.0.0 http://www.osgi.org/xmlns/blueprint/v1.0.0/blueprint.xsd"
-        >
+>
 
     <reference id="transformerMapper" interface="ddf.mime.MimeTypeToTransformerMapper"/>
 
     <reference id="mimeTypeMapper" interface="ddf.mime.MimeTypeMapper"/>
 
-    <reference id="framework" interface="ddf.catalog.CatalogFramework"/>
+    <reference id="framework" interface="ddf.catalog.CatalogFramework" availability="optional"/>
```

Comment:
There seems to be some kind of issue with how the bundles in [catalog-core](https://github.com/codice/ddf/blob/master/catalog/catalog-app/src/main/resources/features.xml#L41) are being started, which seems unique to the [itests.](https://github.com/codice/ddf/blob/master/distribution/test/itests/test-itests-dependencies-app/src/main/filtered-resources/features.xml#L27) The problem is that bundles are waiting for other bundles that are started in that feature to become available, and it doesn't look like we are able to impose an order on them. I've replicating what the itests do outside of pax-exam and everything resolves fine, which makes me think there is a timing issue here.